### ISSUE TITLE
Domain Management: Invisible Options Depending on Color Scheme

### ISSUE THREAD

**Issue by carinapilar** [Automattic/wp-calypso#59373]
### Quick summary

The domain options aren't visible depending on the color scheme you have on your WordPress.com account.

### Steps to reproduce

1. Access your account's settings: https://wordpress.com/me/account
2. Change the color scheme to the Nightfall option
3. Check the Domains page of site with a domain/mapping upgrade
4. Click on the 3-dots icon to see the domain options

### What you expected to happen

To see the menu options.

### What actually happened

It's all black.

### Context

Here's a screenshot:

![DomainsInvisibleOption](https://user-images.githubusercontent.com/16104522/146643098-41b2056c-9cad-4922-88e5-f1abed5dcc8b.jpg)

**Comment by Robertght** [comment]
cc @delputnam

**Comment by delputnam** [comment]
We (Nomado) have a task to address this.

**Comment by leonardost** [comment]
Hi! We just deployed a fix for this in #60149. It should have fixed the problem in other three dots menus as well 🙂 so I'll close this issue now.
